pkg/sql/logictest/testdata/logic_test/sequences

```diff
@@ -2448,20 +2448,6 @@ RESET default_int_size;
 
 subtest end
 
-# The following subtest ensures sequences do not have any unexpected columns
-# on creation. Previously, we were incorrectly creating a rowid column.
-subtest regression_119108
-
-statement ok
-CREATE SEQUENCE sq_119108;
-
-query T
-SELECT column_name from [SHOW COLUMNS FROM sq_119108]
```

Comment:
i think it is still worth keeping this test -- we could have it look at the descriptor proto with something like:
```
select crdb_internal.pb_to_json('desc', descriptor)->'table'->'columns' from system.descriptor where id = 'sq_119108'::regclass::oid;
```